addons/common/fnc_execAfterNFrames.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_execAfterNFrames
+
+Description:
+    Executes the given code after the specified number of frames.
+
+Parameters:
+    _function - The function to run. <CODE>
+    _args     - Parameters passed to the function executing. This will be the same array every execution. (optional, default: []) <ANY>
+    _frames   - The amount of frames the execution of the function should be delayed by. (optional, default: 0) <NUMBER>
+
+Returns:
+    Nothing
```

Comment:
It returns the RV of _function if _frames <= 0. Sorry, pedantry.

Reply:
```suggestion
    Nothing Useful
```